Add release notes fo 4.8.2 (#14740)
docker/docker.github.io: desktop/release-notes.md

---
description: Release notes for Docker Desktop for Mac, Linux, Windows
keywords: Docker desktop, release notes, linux, mac, windows
title: Docker Desktop release notes
toc_max: 2
redirect_from:
- /docker-for-mac/release-notes/
- /docker-for-mac/edge-release-notes/
- /desktop/mac/release-notes/
- /docker-for-windows/edge-release-notes/
- /docker-for-windows/release-notes/
- /desktop/windows/release-notes/
- /desktop/linux/release-notes/
---

This page contains information about the new features, improvements, known issues, and bug fixes in Docker Desktop releases.

> **Note**
>
> The information below is applicable to all platforms, unless stated otherwise.

Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1){: target="_blank" rel="noopener" class="_"} to see what's coming next.

## Docker Desktop 4.8.2

> 2022-05-17

Reviewer: ```suggestion
2022-05-18
```

> Download Docker Desktop 
>

> > [For

Reviewer 2: ```suggestion
> [
```

> Windows](https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64){:
> .button .primary-btn }
> [Mac with Intel chip](https://desktop.docker.com/mac/main/amd64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-amd64){: .button .primary-btn }
> [Mac with Apple
> chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){:
> .button .primary-btn }
> [DEB](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.8.2-amd64.deb?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64){:
> .button .primary-btn }
> [RPM](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.8.2-x86_64.rpm?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64){:
> .button .primary-btn }

> > [Arch package](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.8.2-x86_64.pkg.tar.zst?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64){:

Reviewer 2: Let's remove the 'New' section. Think enough time has passed now 
```suggestion
> [Arch (experimental)](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.8.2-x86_64.pkg.tar.zst?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64){:
```

> .button .primary-btn }

> 

Reviewer: Oh actually we need to mention Compose v2.5.1 here, would the usual
```suggestion

### New

- [Compose v2.5.1](https://github.com/docker/compose/releases/tag/v2.5.1)

```
work?

Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1){: target="_blank" rel="noopener" class="_"} to see what's coming next.

## Docker Desktop 4.8.2
2022-05-17

> Download Docker Desktop 
>
> [For
> Windows](https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64){:
> .button .primary-btn }
> [Mac with Intel chip](https://desktop.docker.com/mac/main/amd64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-amd64){: .button .primary-btn }
> [Mac with Apple
> chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){:
> .button .primary-btn }
> [DEB](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.8.2-amd64.deb?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64){:
> .button .primary-btn }
> [RPM](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.8.2-x86_64.rpm?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64){:

> > .button .primary-btn }

Reviewer: Actually we should probably align the texts across the different platforms? E.g. «For Windows» but «Mac with …» 🤔 
![image](https://user-images.githubusercontent.com/2386884/168761770-59321f18-72b5-4391-90a4-79d315e95e71.png)

> > .button .primary-btn }

Reviewer 2: Have removed 'For' from the Windows button

> [Arch package](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.8.2-x86_64.pkg.tar.zst?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64){:

> > .button .primary-btn }

Reviewer: Do we support arch at the moment? The Linux installation page only lists DEB and RPM: https://docs.docker.com/desktop/linux/install/
@p1-0tr @aiordache?

Reviewer 2: On that same page a little further down it mentions a DD package for Arch based distributions but highlights it's experimental

Reviewer: @christophermclellan any thought?

Reviewer 3: We could change to 'Arch (experimental)' here ?

### Bug fixes and minor changes

- Fixed an issue with manual proxy settings which caused problems when pulling images. Fixes [docker/for-win#12714](https://github.com/docker/for-win/issues/12714) and [docker/for-mac#6315](https://github.com/docker/for-mac/issues/6315).

> - Fixed high CPU usage when extensions are disabled. Fixes [docker/for-mac#6310](https://github.com/docker/for-mac/issues/6310) and [docker/for-win#12266](https://github.com/docker/for-win/issues/12266).

Reviewer 2: ```suggestion
- Fixed high CPU usage when extensions are disabled. Fixes [docker/for-mac#6310](https://github.com/docker/for-mac/issues/6310).
```

> - Docker Desktop now redacts HTTP proxy passwords in log files and diagnostics.

Reviewer: There is no «For all platforms» here, is it intentional because there is no other «For …» section?

Reviewer 2: Yes intentional. There is a note at the top of the page that says 'The information below is applicable to all platforms, unless stated otherwise.'

> - Redact HTTP proxy passwords in log files and diagnostics.

Reviewer 2: ```suggestion
- Docker Desktop now redacts HTTP proxy passwords in log files and diagnostics.
```

- Fixed an issue related to empty remote repositories when the machine is in sleep mode for an extended period of time.
- Fixed a bug where dangling images were not selected in the cleanup process if their name was not marked as "<none>" but their tag is.
- Improved the error message when `docker pull` fails because an HTTP proxy is required.
- Added the ability to clear the search bar easily in Docker Desktop.
- Renamed the "Containers / Apps" tab to "Containers".
- Fixed a silent crash in the Docker Desktop installer when `C:\ProgramData\DockerDesktop` is a file or a symlink.

- Fixed a bug where an image with no namespace, for example `docker pull <private registry>/image`, would be erroneously blocked by Registry Access Management unless access to Docker Hub was enabled in settings.

#### For Mac 

- Docker Desktop's icon now matches Big Sur Style guide. See [docker/for-mac#5536](https://github.com/docker/for-mac/issues/5536)
- Fixed a problem with duplicate Dock icons and Dock icon not working as expected. Fixes [docker/for-mac#6189](https://github.com/docker/for-mac/issues/6189).
- Improved support for the `Cmd+Q` shortcut.

> 
> 

Reviewer: nit
```suggestion

```

- /docker-for-windows/edge-release-notes/
- /docker-for-windows/release-notes/
- /desktop/windows/release-notes/
- /desktop/linux/release-notes/
---

This page contains information about the new features, improvements, known issues, and bug fixes in Docker Desktop releases.

> **Note**
>
> The information below is applicable to all platforms, unless stated otherwise.

Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1){: target="_blank" rel="noopener" class="_"} to see what's coming next.

## Docker Desktop 4.8.2
2022-05-17

> Download Docker Desktop 
>
> [For
> Windows](https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64){:
> .button .primary-btn }
> [Mac with Intel chip](https://desktop.docker.com/mac/main/amd64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-amd64){: .button .primary-btn }
> [Mac with Apple
> chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){:
> .button .primary-btn }

> > [DEB](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.8.2-amd64.deb?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64) |

Reviewer 2: can we add primary button at the end of the linux packages?

> [RPM](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.8.2-x86_64.rpm?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64) |
> [Arch package](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.8.2-x86_64.pkg.tar.zst?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64)

> ### New

Reviewer 2: Let's remove the 'new' section for this release. Not _that_ new any more and has already been repeated on the previous two releases now

Author: Done!
